docs/tutorials/odc-stac.rst

```diff
@@ -0,0 +1,290 @@
+We also use :code:`planetary_computer.sign_inplace` to authorise our connection.
+Type the following into the empty cell below the **Connect to catalog and find items** heading:
+
+.. code-block:: python
+
+   stac_client = Client.open(
+      catalog_query, 
+      modifier=planetary_computer.sign_inplace,
+   )
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+
+Search for items
+^^^^^^^^^^^^^^^^
+
+After setting up the :code:`Client`, we use the :code:`search` method to find items that match our chosen collection, area of interest, and date range.
+Type the following into the empty cell below the **Search for items** heading:
+
+.. code-block:: python
+
+   items = stac_client.search(
+       collections=collections_query,
+       intersects=aoi_geometry,
+       datetime=date_query,
+   ).item_collection()
+
+   print(f"Found {len(items)} items")
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+After running the cell, you should see a printed sentence reporting "Found 17 items"
+
+Troubleshooting
+"""""""""""""""
+
+If the sentence shows a different number of items, try checking whether your :code:`date_query` parameter is correct by printing it:
+
+.. code-block:: python
+
+   print(date_query)
+
+should return :code:`('2017-01-01', '2023-01-01')`.
+If you see a different date range, return to the **Set up query parameters - Date range** section and ensure your :code:`start_date` and :code:`end_date` values match those given in the instructions.
+
+Load items with odc-stac
+------------------------
+
+After producing a list of items to load, we can use the :code:`load` function from :code:`odc-stac` to read the requested assets from the items and return them as xarrays.
+
+Type the following into the empty cell below the **Load items with odc-stac** heading:
+
+.. code-block:: python
+
+   ds = load(
+      items=items,
+      bands=assets_query,
+      geopolygon=aoi_geometry,
+      crs="utm",
+      resolution=30
+   )
+
+   ds
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+You may see warnings appear in red text.
+This is expected behaviour for this dataset and does not mean anything is wrong.
+After running the cell, you should see a :code:`xarray.Dataset` summary.
+
+.. image:: ../_static/tutorial-images/odc-stac/xarray_output.png
+ :width: 600
+ :alt: A table showing a summary of the loaded xarray, including dimensions, coordinates, and data variables.
+ :align: center
+
+Visualise loaded data
+---------------------
+
+To confirm that we have loaded the requested data, it is useful to visualise it. 
+We can use :code:`xarray`'s built-in plotting functionality to make a basic plot.
+
+Type the following into the empty cell below the **Visualise loaded data** heading:
+
+.. code-block:: python
+
+   ds.data.plot.imshow(col="time", col_wrap=3)
```

Comment:
Why the `.data` here? Is this to force computation of the data prior to plotting it? (it's not usually necessary).

Should we include an example of using `.odc.explore()` to quickly plot an interactive map? e.g. `ds.red.isel(time=0).odc.explore()`